### PR TITLE
[TR] Add `Silence`

### PR DESCRIPTION
**Pull request by SStoney6378**
This is the Turkish translation of the Silence segment in the osu! wiki.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Silence/tr.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Susturma
+
+**Susturma** osu! topluluğunun içinde bir kullanıcının başkalarıyla olan iletişimini belli bir derecede limitleyen cezaya verilen terimdir. Susturma işlemini [Küresel Moderasyon Takımı](/wiki/People/Global_Moderation_Team) ve [Aday Gösterme Denetleme Takımı](/wiki/People/Nomination_Assessment_Team) üyeleri yapabilir, ve sadece  [Topluluk Kuralları](/wiki/Rules)nı bozmuş olan kişilere verilir.
```
Comment (ubinici) [inline wiki/Silence/tr.md:3]:
```suggestion
**Susturma**, osu! topluluğunda bir kullanıcının başkalarıyla olan iletişiminin kısıtlanmasına denir. Susturma işlemini [Küresel Moderasyon Takımı](/wiki/People/Global_Moderation_Team) ve [Aday Gösterme Denetleme Takımı](/wiki/People/Nomination_Assessment_Team) üyeleri yapabilir, ve susturma yalnızca [Topluluk Kurallarına](/wiki/Rules) uymayan kullanıcılara verilir.
```

File: wiki/Silence/tr.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Sınırlamalar
+
+Moderatörler tarafından susturulmuş olan kullanıcılar aşağıdakileri yapamazlar:
```
Comment (ubinici) [inline wiki/Silence/tr.md:7]:
```suggestion
Moderatörler tarafından susturulmuş olan kullanıcılar aşağıdaki işlemleri yapamazlar:
```

File: wiki/Silence/tr.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Hem oyun içinde hem de site üzerinden [Sohbet](/wiki/Client/Interface/Chat_console) özelliği (Genel ve Özel),
```
Comment (ubinici) [inline wiki/Silence/tr.md:9]:
```suggestion
- Hem oyun içerisinde hem de web sitesi üzerinden genel ve özel [sohbet](/wiki/Client/Interface/Chat_console) kanallarına erişim, 
```

File: wiki/Silence/tr.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+**Susturma** osu! topluluğunun içinde bir kullanıcının başkalarıyla olan iletişimini belli bir derecede limitleyen cezaya verilen terimdir. Susturma işlemini [Küresel Moderasyon Takımı](/wiki/People/Global_Moderation_Team) ve [Aday Gösterme Denetleme Takımı](/wiki/People/Nomination_Assessment_Team) üyeleri yapabilir, ve sadece  [Topluluk Kuralları](/wiki/Rules)nı bozmuş olan kişilere verilir.
+
+## Sınırlamalar
+
+Moderatörler tarafından susturulmuş olan kullanıcılar aşağıdakileri yapamazlar:
+
+- Hem oyun içinde hem de site üzerinden [Sohbet](/wiki/Client/Interface/Chat_console) özelliği (Genel ve Özel), 
+- Herhangi bir osu! forumunda gönderi atma
+- Sitede herhangi bir yere yorum bırakma (haberler, değişiklik günlüğü, beatmapler)
```
Comment (ubinici) [inline wiki/Silence/tr.md:11]:
```suggestion
- Web sitesinde herhangi bir yere yorum bırakma (haberler, değişiklik kayıtları, beatmapler)
```

File: wiki/Silence/tr.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+   [Beatmap Tartışmaları](/wiki/Beatmap_discussion)nda yer alma
```
Comment (ubinici) [inline wiki/Silence/tr.md:12]:
```suggestion
   [Beatmap tartışmalarında](/wiki/Beatmap_discussion) yer alma
```

File: wiki/Silence/tr.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+- Profil detaylarını düzenleme (Örn. profil fotoğrafı, "ben!" kısmı, vb.)
+- Beatmapleri güncelleme ve yükleme
+- [Çok Oyunculu](/wiki/Client/Interface/Multiplayer) oyunlara katılma
```
Comment (ubinici) [inline wiki/Silence/tr.md:15]:
```suggestion
- [Çok oyunculu](/wiki/Client/Interface/Multiplayer) oyunlara katılma
```

File: wiki/Silence/tr.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+Bunlara ek olarak, susturulan kişilerin bütün genel konuşma kanallarındaki ve özel konuşmalardaki mesajları kaldırılır.[^chat-cleanup]
```
Comment (ubinici) [inline wiki/Silence/tr.md:17]:
```suggestion
Bunlara ek olarak, susturulan kullanıcılara ait mesaj kayıtları tüm genel ve özel sohbet kanallarından kaldırılır.[^chat-cleanup]
```

File: wiki/Silence/tr.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+Oyuncu susturulduktan sonra, yazı yazma yerinde bir uyarı çıkar ve susturmanın ne zaman biteceğini gösterir. Susturma kaydı sitede profilin en altında görülebilir ve 28 gün sonra kaybolur.
+
+## Susturmaya karşı itiraz
+
+Eğer haksız yere susturulduğunuzu düşünüyorsanız, [Hesap Destek Ekibi](/wiki/People/Account_support_team#accounts@ppy.sh)ne [accounts@ppy.sh](mailto:accounts@ppy.sh) mailinden ulaşıp itiraz edin. osu! hesabınızın linkli olduğu emailden mesajı göndermeye dikkat edin; osu! kullanıcı adınızı söyleyin ve yaklaşık hangi tarihte ne zaman susturulduğunuzu söyleyin.
```
Comment (ubinici) [inline wiki/Silence/tr.md:23]:
```suggestion
Eğer haksız yere susturulduğunuzu düşünüyorsanız, [hesap destek ekibiyle](/wiki/People/Account_support_team#accounts@ppy.sh) [accounts@ppy.sh](mailto:accounts@ppy.sh) adresi üzerinden iletişime geçebilirsiniz. İtirazınızı osu! hesabınızın linkli olduğu e-posta adresinden göndermeye dikkat edin; e-posta içeriğinde osu! kullanıcı adınızın ve yaklaşık hangi tarihte ne zaman susturulduğunuzun bilgisinin bulunduğundan emin olun.
```

File: wiki/Silence/tr.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+## Susturulmak için genel sebepler
```
Comment (ubinici) [inline wiki/Silence/tr.md:25]:
```suggestion
## Genel susturma sebepleri
```

File: wiki/Silence/tr.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+*Dikkat: Özel mesajlarda istemeyen veya rıza göstermeyen kişilere söylenmediği sürece "uygunsuz davranışlar"da bulunmak kurallara aykırı değildir. Eğer rapor edilirse, uygun cezalandırma yapılacaktır.*
```
Comment (ubinici) [inline wiki/Silence/tr.md:27]:
```suggestion
*Dikkat: Özel mesajlardaki "uygunsuz davranışlar", alıcının rızası veya izni dahilinde olduğu sürece, kurallara tabi değildir. Ancak raporlandığı takdirde uygun cezalandırma yapılacaktır.*
```

File: wiki/Silence/tr.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+- Genel sohbette veya forumlarda (Örn. `#multiplayer`, beatmap tartışmaları, yorumlar, vb.) uygunsuz davranışlar, buna ek olarak — bununla limitli olmamakla beraber — spamlamak, anlamsız sohbetler, ırkçılık. vb.
+- Uygunsuz herhangi bir içeriği yüklemek veya paylaşmak (Örn. profil fotoğrafları, beatmap paylaşımları, profil detayları, kullanıcı biyografisi, vb.
```
Comment (ubinici) [inline wiki/Silence/tr.md:30]:
```suggestion
- Uygunsuz herhangi bir içeriği yüklemek veya paylaşmak (Örn. profil fotoğrafları, beatmap paylaşımları, profil detayları, kullanıcı bilgileri, vb.)
```

File: wiki/Silence/tr.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+## Susturulmak için genel sebepler
+
+*Dikkat: Özel mesajlarda istemeyen veya rıza göstermeyen kişilere söylenmediği sürece "uygunsuz davranışlar"da bulunmak kurallara aykırı değildir. Eğer rapor edilirse, uygun cezalandırma yapılacaktır.*
+
+- Genel sohbette veya forumlarda (Örn. `#multiplayer`, beatmap tartışmaları, yorumlar, vb.) uygunsuz davranışlar, buna ek olarak — bununla limitli olmamakla beraber — spamlamak, anlamsız sohbetler, ırkçılık. vb.
+- Uygunsuz herhangi bir içeriği yüklemek veya paylaşmak (Örn. profil fotoğrafları, beatmap paylaşımları, profil detayları, kullanıcı biyografisi, vb. 
+
+Susturmanın uzunluğu yapılan ihlalin ciddiyetine göre değişebilir. Susturmalar üst üste binebilir ve cezası bir öncekine göre daha uzun olur. Eğer bir suç veya ihlal yeterince ağırsa veya oyuncu durmadan kuralları çiğniyorsa, ceza susturma olmak yerine [Hesap Kısıtlama](/wiki/Help_centre/Account_restrictions) ile sonuçlanabilir.
```
Comment (ubinici) [inline wiki/Silence/tr.md:32]:
```suggestion
Susturmanın uzunluğu yapılan ihlalin ciddiyetine göre değişebilir. Susturmalar katlanarak artar, böylelikle süresi bir öncekine göre daha uzun olur. Eğer bir suç veya ihlal yeterince ağırsa veya oyuncu durmadan kuralları çiğniyorsa, uygulanacak ceza, susturulma yerine [hesabın kısıtlanması](/wiki/Help_centre/Account_restrictions) ile sonuçlanabilir.
```

File: wiki/Silence/tr.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+## Referanslar
+
+[^chat-cleanup]: [Blog post by ppy (2012-12-17) "This Week in osu!"](https://blog.ppy.sh/post/38114063519/this-week-in-osu-5)
```
Comment (ubinici) [inline wiki/Silence/tr.md:36]:
```suggestion
[^chat-cleanup]: [ppy tarafından yayımlanan bir blog yazısı (2012-12-17) "This Week in osu!"](https://blog.ppy.sh/post/38114063519/this-week-in-osu-5)
```

**Review by ubinici — CHANGES_REQUESTED** [review]
Native review en route.

**Review by TicClick — APPROVED** [review]

**Pull request merged**